<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>518</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radio Button</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnVerif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Cores</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radVerm">
           <property name="text">
            <string>Vermelho</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radVerd">
           <property name="text">
            <string>Verde</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radAzul">
           <property name="text">
            <string>Azul</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Transportes</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radAv">
           <property name="text">
            <string>Avi√£o</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radCar">
           <property name="text">
            <string>Carro</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radNav">
           <property name="text">
            <string>Navio</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Linguagens</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radCpp">
           <property name="text">
            <string>C++</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radCsh">
           <property name="text">
            <string>C#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radJav">
           <property name="text">
            <string>Java</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>10</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mudar Imagem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="itNov"/>
    <addaction name="itEdi"/>
    <addaction name="itSalv"/>
    <addaction name="separator"/>
    <addaction name="itSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="itPesq"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <action name="itNov">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/icons/icones/user_go.png</normaloff>:/icons/icones/user_go.png</iconset>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="itEdi">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/icons/icones/user_edit.png</normaloff>:/icons/icones/user_edit.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="toolTip">
    <string>Editar</string>
   </property>
  </action>
  <action name="itSalv">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/icons/icones/user_add.png</normaloff>:/icons/icones/user_add.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="itSair">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/icons/icones/door_out.png</normaloff>:/icons/icones/door_out.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="itPesq">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/icons/icones/zoom.png</normaloff>:/icons/icones/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
